ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1152,7 +1152,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_ChatAndNotificationsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
```

Comment:
This is a fine way to align the buttons.  You may want less left margin on the button, so that it shows up right after the textbox.